doc/how-to-ota.md

```diff
@@ -0,0 +1,41 @@
+# How To Release a Gateway Update
+
+1. Bump the [semantic version](http://semver.org/) in [package.json](https://github.com/mozilla-iot/gateway/blob/master/package.json)
+2. Push the bump to master
+3. Make a new clone of this repo *on a raspberry pi*
+4. Archive this new clone as a tar.gz. For example, if you cloned it as
+   "gateway", this command should do the trick: `tar czvf gateway.tar.gz
+   gateway`
+5. Run `yarn install` in the new clone
+6. Move `gateway.tar.gz` into the clone
+7. Run [tools/create-release-archives.sh](https://github.com/mozilla-iot/gateway/blob/master/tools/create-release-archives.sh), producing `gateway-<hash>.tar.gz`
+   and `node_modules-<hash>.tar.gz`
+8. Create a release using the version from step 1 as its tag name
+9. Attach the release archive to this release
+10. Publish the release when ready or save as a draft
+
```

Comment:
Presumably we could create a script which automates the above?

Reply:
Yes, this is definitely a place where a script would help out